README.md

```diff
@@ -53,6 +53,19 @@ on Red Hat Enterprise Linux.
     - Because the VM was left running after the first context manager block.
     - Fortunately, no such test currently exists (the use case is rare).
 
+- `CONTEST_DATASTREAM`
+  - Specify a filesystem path to a datastream XML to use for testing.
+
+- `CONTEST_PLAYBOOK`
+  - Specify a filesystem path to an Ansible YAML file to use as a playbook
+    for testing.  
+    A magical string of '{PROFILE}' expands to a profile name being tested.
+
+- `CONTEST_KICKSTART`
+  - Specify a filesystem path to an configuration file to use as an Anaconda
+    kickstart for testing.  
+    A magical string of '{PROFILE}' expands to a profile name being tested.
```

Comment:
Magical string for distro major version , `DISTRO_MAJOR`, would be also useful.

Reply:
I thought about it as well, but it's not as straightforward.

I decided not to do it, because it would make the neat
```python
return Path(override.format(PROFILE=profile))
```
a lot longer, but mainly because you can already anticipate the distro major version before you run the tests, whereas, without `{PROFILE}`, you would have to run each test separately.

Finally, you already need to hardcode distro major version when running `tmt`, because of ie. `-c distro=rhel9` as context.

If you think it would still be useful despite these shortcomings, I can add it. Probably for datastream too then.

Reply:
The TMT context point convinced me. Let's leave it as it is without distro major version.